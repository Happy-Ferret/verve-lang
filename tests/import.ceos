import { square } from "./import_helper_1"
/*import * from "./import_helper_2" as Helper*/

print(square(2))
/*print(Helper#add(40, 2))*/
/*print(Helper#sub(40, 2))*/
