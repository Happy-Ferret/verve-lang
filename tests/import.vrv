import { square } from "./import_helper_1"
import * from "./import_helper_2"
import * from "./import_helper_3" as Helper

print(square(2))
print(add(40, 2))
print(sub(40, 2))
print(Helper#mul(40, 2))
print(mul(40, 2))
