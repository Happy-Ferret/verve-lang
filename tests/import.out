4
42
38
80
